src/v/cluster/tests/BUILD

```diff
@@ -253,3 +314,233 @@ redpanda_cc_gtest(
         "@seastar",
     ],
 )
+
+redpanda_cc_btest(
+    name = "partition_allocator_tests",
+    timeout = "short",
+    srcs = [
+        "partition_allocator_tests.cc",
+    ],
+    deps = [
+        "//src/v/cluster",
+        "//src/v/cluster/tests:partition_allocator_fixture",
+        "//src/v/model",
+        "//src/v/raft",
+        "//src/v/raft:fundamental",
+        "//src/v/random:fast_prng",
+        "//src/v/random:generators",
+        "//src/v/test_utils:seastar_boost",
+        "@abseil-cpp//absl/container:flat_hash_set",
+        "@boost//:test",
+        "@seastar",
+        "@seastar//:testing",
+    ],
+)
+
+redpanda_cc_btest(
+    name = "partition_balancer_planner_test",
+    timeout = "short",
+    srcs = [
+        "partition_balancer_planner_test.cc",
+    ],
+    deps = [
+        "//src/v/base",
+        "//src/v/cluster",
+        "//src/v/cluster/tests:partition_balancer_planner_fixture",
+        "//src/v/test_utils:seastar_boost",
+        "//src/v/utils:stable_iterator",
+        "@boost//:test",
+        "@seastar",
+        "@seastar//:testing",
+    ],
+)
+
+redpanda_cc_btest(
+    name = "simple_batch_builder_test",
```

Comment:
I wonder if any of these tests 
have -c1 set in cmake and we should mirror that to Bazel so we have higher concurrency in the test runner

Reply:
I don't think these particular tests have -c1

https://github.com/redpanda-data/redpanda/blob/dev/src/v/cluster/tests/CMakeLists.txt#L40-L49